### DISCUSSION TITLE
[Newbie question] Nous A1Z do not want to operate as a router

### DISCUSSION THREAD

**Discussion by EriHau** [Koenkk/zigbee2mqtt#21355] [Devices]
Hi, I'm a bit confused: I'm just starting to set up my Zigbee network and have bought 2x  A1Z (Model: TS011F , Manufacturer: _TZ3000_2putqrmw) to use as routers. But I can't manage to connect new devices via these routers. They can only be registered on the coordinator. I have upgraded Z2M and the coordinator to the latest version. No change. 
When I run a coordinator_check, I always get such a message:

`{"data":{"missing_routers":[{"friendly_name":"Vorzimmer","ieee_address":"0xa4c138b65fd735b5"},{"friendly_name":"Wohnzimmer","ieee_address":"0xa4c138d0f0405baf"}]},"status":"ok"}`
Does this mean that both are not acting as router?

The two plugs work perfectly apart from that. What can I do next? Or do my A1Z not have a router function?
Thanks !

**Comment by crystal-way** [comment]
Have you solved this issue?
Does A1Z have routing function or only works as end point?
